### PR TITLE
lunacy: fix manifest

### PR DESCRIPTION
**Pull request by xeijin**

### CODE REVIEW

File: bucket/lunacy.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "innosetup": true,
     "checkver": {
         "url": "https://docs.icons8.com/release-notes/",
-        "re": ">([\\d.]+)</h2>"
+        "re": ">(\\d+\\.\\d+\\.\\d+).*<\/h2>"
```
Comment (Ash258) [inline bucket/lunacy.json]:
```suggestion
        "regex": ">([\\d.]+).*?<\/h2>"
```

Comment (xeijin) [reply bucket/lunacy.json]: @Ash258 done

Comment (Ash258) [reply bucket/lunacy.json]: Apply everything from details to fix installation

**Comment by xeijin** [comment]
@Ash258 the fix seems a bit crude ... why is this happening in the first place? Some issue with `innoup`?

Also apologies if I'm missing something but I find GitHub's pull request interface completely unintuitive, what would you like me to do?

**Comment by Ash258** [comment]
This is usual flow of innosetup with both architectures. `,2` files are 32bit and other 64bit.

Fix manifest with new insstaller scripts 

A

**Comment by Ash258** [comment]
@xeijin I do not know what I can do to make it more clear. Bellow you have fixed manifest with handling multiple architectures in same inno based archive. Apply it and everything will be working 👀

https://github.com/lukesampson/scoop-extras/pull/2426#pullrequestreview-258602407
https://github.com/lukesampson/scoop-extras/pull/2426#pullrequestreview-258599088

```json
{
    "version": "4.0.3",
    "description": "Free Graphic Design Software.",
    "homepage": "https://icons8.com/lunacy",
    "license": {
        "identifier": "Proprietary",
        "url": "https://docs.icons8.com/lunacy%20license.md"
    },
    "url": "https://desktop.icons8.com/lunacy/LunacySetup_4.0.3.exe",
    "hash": "fe8177bd182584b83ece7b564a67791b57333f5c9e395db8f871fff039c64df2",
    "innosetup": true,
    "architecture": {
        "64bit": {
            "installer": {
                "script": [
                    "Get-ChildItem \"$dir\" '*,1.*' | ForEach-Object { Rename-Item $_.Fullname \"$($_.Basename.TrimEnd(',1'))$($_.Extension)\" }",
                    "Remove-Item \"$dir\\*,2*\""
                ]
            }
        },
        "32bit": {
            "installer": {
                "script": [
                    "Get-ChildItem \"$dir\" '*,2.*' | ForEach-Object { Rename-Item $_.Fullname \"$($_.Basename.TrimEnd(',2'))$($_.Extension)\" }",
                    "Remove-Item \"$dir\\*,1*\""
                ]
            }
        }
    },
    "bin": "Lunacy.exe",
    "shortcuts": [
        [
            "Lunacy.exe",
            "Lunacy"
        ]
    ],
    "checkver": {
        "url": "https://docs.icons8.com/release-notes/",
        "regex": ">([\\d.]+).*</h2>"
    },
    "autoupdate": {
        "url": "https://desktop.icons8.com/lunacy/LunacySetup_$version.exe"
    }
}
```

**Pull request closed without merging**